views/support/faq.tt

```diff
@@ -0,0 +1,56 @@
+[%# faq.tt
+
+Support FAQ page
+
+Authors:
+    hotlevel4 <hotlevel4@hotmail.com>
+
+This program is free software; you may redistribute it and/or modify it under
+the same terms as Perl itself. For a copy of the license, please reference
+'perldoc perlartistic' or 'perldoc perlgpl'.
+
+%]
+
+[%- CALL dw.active_resource_group( "foundation" ) -%]
+
+[%- sections.title = '.title' | ml -%]
+
+<p>[% '.title.text2' | ml( sitenameshort = site.nameshort ) %]</p>
+<p>[% '.search.text' | ml %]</p>
+
+<form method='GET' action='[% site.root %]/support/faqsearch'>
+<div class="row">
+  <div class="large-12 columns">
+    <div class="row collapse">
+      <div class="small-10 columns">
+        <input type="text" placeholder="Search terms" name="q">
```

Comment:
Nice. I'd suggest using "type=search" here for semantics (won't affect appearance, just behavior in e.g., screenreaders)